README.md

```diff
@@ -42,7 +42,7 @@ epilogue.initialize({
 });
 
 // Create REST resource
-var userResource = epilogue.resource({
+var users = epilogue.resource({
```

Comment:
as long as we're going for clarity and consistency, I would prefer calling these "userResource", as the singular "user" is generally used to indicate the model being used. Otherwise LGTM!